docs/website/static/.circleci/config.yml

```
@@ -0,0 +1,10 @@
+# CircleCI 2.0 Config File
+# This config file will prevent tests from being run on the gh-pages branch.
+version: 2
+jobs:
+  build:
+    machine: true
```

This job can be speed up by using docker instead of machine.

```suggestion
    docker:
      - image: busybox:1.31.0
```